semgrep/tests/conftest.py

```diff
@@ -85,8 +85,6 @@ def _clean_output_json(output_json: str) -> str:
             p = r.get("path")
             if p and "/tmp" in p:
                 r["path"] = "/tmp/masked/path"
-                # the fingerprint contains the path too
-                r["extra"]["fingerprint"] = "0x42"
```

Comment:
Nice! Ok to remove since with the change in this PR fingerprint should be stable